test/e2e/user_workload_monitoring_test.go

```diff
@@ -860,6 +864,141 @@ func assertTenancyForRules(t *testing.T) {
 	}
 }
 
+func assertTenancyForLabels(t *testing.T) {
+	const testAccount = "test-labels"
+
+	_, err := f.CreateServiceAccount(userWorkloadTestNs, testAccount)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Grant enough permissions to read labels. todo: check if need create new role
```

Comment:
The labels endpoints are similar to the query endpoints: if you are authorized for /api/v1/query, you should also be for /api/v1/labels.

Reply:
can you remove the `todo: ...` part (or address it)?

Reply:
yup. already checked: no need for another account